IdentityCore/src/cache/token/MSIDPRTCacheItem.m

```diff
@@ -48,6 +49,7 @@ - (id)initWithCoder:(NSCoder *)decoder
     if (self)
     {
         self.sessionKey = [decoder decodeObjectOfClass:[NSString class] forKey:@"sessionKey"];
+        self.deviceID = [decoder decodeObjectOfClass:[NSString class] forKey:@"deviceID"];
```

Comment:
please also update the schema document with this property

Reply:
Will do.